### DISCUSSION TITLE
This user account is blocked. ERROR

### DISCUSSION THREAD

**Discussion by ivp9** [vercel/next.js#57577] [Help]
### Summary

How to fix "This user account is blocked" error? I sent the email to registration@vercel.com, but nobody answer for a 2 weeks... My case number is 00164473.

![2023-10-27_11-38-27](https://github.com/vercel/next.js/assets/55412852/50a38e42-6121-491e-9c66-9723b4d75283)
![2023-10-09_19-00-43 (1)](https://github.com/vercel/next.js/assets/55412852/5b26a3fb-ba00-40ef-979b-3855d81e0a79)

**Comment by idegin** [comment]
I'm having same issue
